docs/variants/dell_optiplex/building-manual.md

```diff
@@ -32,9 +32,12 @@ To build coreboot image, follow the steps below:
     $ docker run --rm -it \
        -v $PWD:/home/coreboot/coreboot \
        -w /home/coreboot/coreboot \
-       coreboot/coreboot-sdk:0ad5fbd48d /bin/bash
+       coreboot/coreboot-sdk:VERSION /bin/bash
     ```
 
+    * `VERSION` should be replaced according to version you building:
+      - `v0.1.0` - `2021-09-23_b0d87f753c`
```

Comment:
This might be too new for 4.12 coreboot.

Reply:
@miczyg1 why are you talking about coreboot 4.12? All stuff for Dell OptiPlex is rebased to most recent version of coreboot. Isn't this what we agreed to internally?

Reply:
Ahh forgot... NVM.